yarn-audit-known-issues

```diff
@@ -0,0 +1,5 @@
+{"type":"auditAdvisory","data":{"resolution":{"id":1544,"path":"@hmcts/ccd-case-ui-toolkit>@angular/cli>pacote>npm-registry-fetch","dev":false,"optional":false,"bundled":false},"advisory":{"findings":[{"version":"3.9.1","paths":["@hmcts/ccd-case-ui-toolkit>@angular/cli>pacote>npm-registry-fetch"]}],"id":1544,"created":"2020-07-07T19:05:44.803Z","updated":"2020-07-07T19:05:50.413Z","deleted":null,"title":"Sensitive Data Exposure","found_by":{"link":"","name":"Unknown","email":""},"reported_by":{"link":"","name":"Unknown","email":""},"module_name":"npm-registry-fetch","cves":[],"vulnerable_versions":"<4.0.5 || >=5.0.0 <8.1.1","patched_versions":">=4.0.5 <5.0.0 || >=8.1.1","overview":"Affected versions of npm-registry-fetch are vulnerable to an information exposure vulnerability through log files.   The package supports URLs like <protocol>://[<user>[:<password>]@]<hostname>[:<port>][:][/]<path>. The password value is not redacted and is printed to stdout and also to any generated log files.","recommendation":"Upgrade to version 4.0.5, 8.1.1 or later.","references":"- [GitHub Advisory](https://github.com/npm/npm-registry-fetch/security/advisories/GHSA-jmqm-f2gx-4fjv)","access":"public","severity":"low","cwe":"CWE-532","metadata":{"module_type":"","exploitability":3,"affected_components":""},"url":"https://npmjs.com/advisories/1544"}}}
```

Comment:
Should be able to delete this file after fixing the issues.

Reply:
Have fixed the vulnerabilities